docs/sidebars.js

```diff
@@ -66,6 +66,11 @@ const sidebars = {
           label: 'Integrate Within a Framework',
           id: 'content/getting-started/framework',
         },
+        {
+          type: 'doc',
+          label: 'Production Environment',
```

Comment:
I'd recommend "Production Checklist"

Reply:
I would lean towards "Production Best-Practices" as it is strictly a checklist.  More as a recommendation / guide.

Reply:
Sounds great 👍 "Product Best-Practices" it is!